storage/src/test/java/com/tencent/rss/storage/handler/impl/HdfsFileReaderTest.java

```diff
@@ -91,166 +89,7 @@ public void readDataTest() throws IOException {
-      for (int i = 0; i < segments.length; ++i) {
-        writer.writeIndex(segments[i]);
-      }
-
-      int[] data = {1, 3, 5, 7, 9};
-
-      ByteBuffer buf = ByteBuffer.allocate(4 * data.length);
-      buf.asIntBuffer().put(data);
-      writer.writeData(buf.array());
-    }
-
-    try (HdfsFileReader reader = new HdfsFileReader(path, conf)) {
-      // test limit
-      int limit = 10;
-      thrown.expect(IllegalStateException.class);
-      thrown.expectMessage("Invalid index file");
-      reader.readIndex(limit);
-    }
-  }
-
-  @Test
-  public void readUploadFileHeaderTest() {
-    try {
-      Path path = new Path(HDFS_URI, "readUploadFileHeaderTest");
-      List<Integer> partitionList = Lists.newArrayList(1, 2, 3);
-      List<Long> sizeList = Lists.newArrayList(1L, 2L, 3L);
-      List<Long> fileSizeList = Lists.newArrayList(1L, 2L, 3L);
-      try (HdfsFileWriter writer = new HdfsFileWriter(path, conf)) {
-        writer.writeHeader(partitionList, sizeList, fileSizeList);
-      }
-      try (HdfsFileReader reader = new HdfsFileReader(path, conf)) {
-        ShuffleIndexHeader header = reader.readHeader();
-        assertEquals(partitionList.size(), header.getPartitionNum());
-        assertEquals(partitionList.size(), header.getIndexes().size());
-        for (int i = 0; i < header.getPartitionNum(); i++) {
-          ShuffleIndexHeader.Entry entry = header.getIndexes().get(i);
-          assertEquals(partitionList.get(i), entry.getPartitionId());
-          assertEquals(sizeList.get(i), entry.getPartitionIndexLength());
-          assertEquals(fileSizeList.get(i), entry.getPartitionDataLength());
-        }
-      }
-
-      Path failPath = new Path(HDFS_URI, "readUploadFileHeaderFailTest");
-      try (HdfsFileWriter writer = new HdfsFileWriter(failPath, conf)) {
-        ByteBuffer headerContentBuf = ByteBuffer.allocate(4 + 4 + 8 + 4 + 4);
-        headerContentBuf.putInt(1);
-        headerContentBuf.putInt(1);
-        headerContentBuf.putLong(1);
-        headerContentBuf.putLong(1);
-        headerContentBuf.flip();
-        writer.writeData(headerContentBuf);
-      }
-      try (HdfsFileReader reader = new HdfsFileReader(failPath, conf)) {
-        boolean isException = false;
-        try {
-          ShuffleIndexHeader header = reader.readHeader();
-        } catch (IOException ioe) {
-          isException = true;
-        }
-        assertTrue(isException);
-      }
-    } catch (Exception e) {
-      e.printStackTrace();
-      fail();
-    }
-  }
-
-  @Test
-  public void bigIndexDataTest() {
-    try {
-      String basePath = HDFS_URI + "test_big_data/";
-      HdfsFileWriter writer = new HdfsFileWriter(new Path(basePath + "1.index"), conf);
-      List<Integer> indexes = Lists.newArrayList();
-      List<Long> sizes = Lists.newArrayList();
-      List<Long> fileSizes = Lists.newArrayList();
-      for (int i = 0; i < 1024; i++) {
-        indexes.add(i);
-        sizes.add(10240L);
```

Comment:
Where were these cases moved?

Reply:
The HdfsFileReader is now only deal with pure data reading. All related test cases are removed.

Reply:
I think we shouldn't delete these cases when refactor.

Reply:
These methods and logic do not exist any more. What't the point to keep theirs ut?

Reply:
Not Covered. Where do the code cover the large size index case?

Reply:
I don't think we need it. All index parsing logic is covered.